src/main/java/com/notably/logic/commands/NewCommand.java

```diff
@@ -14,6 +17,7 @@
     public static final String COMMAND_WORD = "new";
     public static final String COMMAND_SHORTHAND = "n";
     private final Block toAdd;
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `NewCommand`!